README.md

```diff
@@ -112,15 +112,11 @@ To check if data is loaded use the
 ```javascript
 import React, { Suspense } from 'react';
 
-const MyComponent = () => {
-  return (
-    return (
-      <Suspense fallback={<div>Loading...</div>}>
-        <Dogs />
-      </Suspense>
-    )
-  );
-}
+const MyComponent = () => (
```

Comment:
Could you please change it to

```
const MyComponent = () => {
  return (
    <Suspense fallback={<div>Loading...</div>}>
      <Dogs />
    </Suspense>
  );
}
```

?

I haven't seen a component written like
```const MyComponent = () => (<SomeJsx />)```
in my codebase yet. :)

Reply:
Sorry, I'm confused, because the very first example in the readme is this:
```js
const App = () => (
  <ApolloProvider client={client}>
    <MyRootComponent />
  </ApolloProvider>
);
```

Reply:
Haha, good catch. Se let's leave it that way. I think I copied it from pupstagram example. Thanks :)